apps/accounts/views/oauth2_profile.py

```diff
@@ -37,7 +39,9 @@ def get_userinfo(user):
 @protected_resource()
 def openidconnect_userinfo(request, **kwargs):
     if request.path.startswith('/v2') and (not waffle.flag_is_active(request, 'bfd_v2_flag')):
-        raise exceptions.NotFound("bfd_v2_flag not active.")
+        err = exceptions.NotFound("bfd_v2_flag not active.")
```

Comment:
Couldn't we tell which ones were being blocked because they didn't have the flag turned on by this exception message? "bfd_v2_flag not active."

Reply:
IIRC exceptions are not logged so there's no way to search for that string in splunk otherwise.

Reply:
yea, that's the reason this logging is introduced.